.circleci/config.yml

```diff
@@ -92,9 +92,18 @@ jobs:
           name: terraform init & apply
           working_directory: test/acceptance/setup-terraform
           command: |
+            set -x
+
             terraform init
 
-            terraform apply -var tags="{\"build_url\": \"$CIRCLE_BUILD_URL\"}" -auto-approve -var launch_type=<<parameters.launch_type>>
+            VARS="-var tags={\"build_url\":\"$CIRCLE_BUILD_URL\"}"
+            VARS+=' -var launch_type=FARGATE'
+            case $CIRCLE_BRANCH in
+                main | release/*) VARS+=" -var enable_hcp=true";;
+                *) VARS+=" -var enable_hcp=false";;
+            esac
```

Comment:
I don't love this but I didn't find a better way to alter the command based on the git branch.